pandas/core/internals/blocks.py

```diff
@@ -925,26 +925,28 @@ def putmask(
         inplace: bool = False,
         axis: int = 0,
         transpose: bool = False,
-    ):
+    ) -> List["Block"]:
         """
         putmask the data to the block; it is possible that we may create a
         new dtype of block
 
-        return the resulting block(s)
+        Return the resulting blocks.
 
         Parameters
         ----------
-        mask  : the condition to respect
+        mask : the condition to respect
         new : a ndarray/object
-        align : boolean, perform alignment on other/cond, default is True
-        inplace : perform inplace modification, default is False
+        align : bool, default True
+            Perform alignment on other/cond.
+        inplace : bool, default False
+            Perform inplace modification.
         axis : int
-        transpose : boolean
-            Set to True if self is stored with axes reversed
+        transpose : bool, default False
+            Set to True if self is stored with axes reversed.
 
         Returns
         -------
-        a list of new blocks, the result of the putmask
+        List[Block]
```

Comment:
from numpdoc for Returns section... 

"Explanation of the returned values and their types. Similar to the Parameters section, except the name of each return value is optional. The type of each return value is always required"

the pandas docstring guide appears to say the same.

The guides do NOT say this section is optional.

so to comply something like...
```suggestion
        list of Block
            A list of new blocks, the result of the putmask.
```

personally, i'd be happy following the google style here.

"Describe the type and semantics of the return value. If the function only returns None, this section is not required. It may also be omitted if the docstring starts with Returns or Yields (e.g. """Returns row from Bigtable as a tuple of strings.""") and the opening sentence is sufficient to describe return value."

and make the returns section optional for internal docstings in the validation.

Reply:
we don't use google style anywhere, this is quite descriptive.